### ISSUE TITLE
scipy.misc.imresize removed in scipy 1.3.0.

### ISSUE THREAD

**Issue by cgo** [anuragranj/cc#14]
imresize is no longer provided by scipy. If the intention is to support 1.3.0 and above, I suggest to switch to Pillow or something else.

**Comment by anuragranj** [comment]
Thanks @cgo . I can start working on it, but if you already have something, could you send a pull request?

**Comment by cgo** [comment]
I just downgraded scipy on my end :-)  just wanted to give CC a try.
On that note, I'll contact you with another question that has nothing to do with this issue; stay tuned :)

**Comment by anuragranj** [comment]
We now have a bleeding edge version that uses PIL instead of skimage. It is available on this branch.

https://github.com/anuragranj/cc/tree/pil

Please let me know if this works, or if there are any issues with it.

**Comment by cgo** [comment]
Hey @anuragranj, thanks for the change; I noticed it was re-opened, anything I can do to check this?

**Comment by anuragranj** [comment]
Yes, I am keeping this open until we are sure that this works. Could you try the pil branch and see if there are any problems?

**Comment by ShristiDasBiswas** [comment]
> Yes, I am keeping this open until we are sure that this works. Could you try the pil branch and see if there are any problems?

hi! does it work with pil?

**Comment by anuragranj** [comment]
To my knowledge, it worked at the time.
